app/controllers/write_offs_controller.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class WriteOffsController < ApplicationController
```

Comment:
I think we'd talked about using AdminFormsController for this last week – did it not work out? Or are we planning to do some separate refactoring?

I would be tempted to rename this to `WriteOffFormsController` as I think all the other controllers using form objects include `Form` in the name. So it might make it more consistent and be a good indicator that it uses a form object.

Reply:
We did talk about the AdminFormsController in regard to transient payments, no refunds. I am not convinced about using it here yet. Currently, here I don't use anything of the things implemented in AdminFormsController. There is still a lot of code there that is dependant on renewals, and I need to refactor that out first.

Reply:
Ahhh, that was it – I was thinking of the wrong feature 🤦‍♀ Ignore me then!